rest/handler.go

```diff
@@ -189,6 +189,10 @@ func (h *handler) invoke(method handlerMethod) error {
 
 	h.logRequestLine()
 
+	if base.LogKeys["HTTP++"] {
```

Comment:
There are other places we do a non-safe read of that map - if you add a new method ('LogKeyExplicitlyEnabled'?  'AdminAskedForThisLogKeySpecifically'?) to logger, you might want to use it in those places as well.  I think we've avoided races thus far because we update that map so rarely, but I've seen a few race errors in unit tests when the tests try to set log keys.